src/modules/authentication/components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,7 +1,105 @@
+import styles from "./CSS/ChangePassword.module.css"
+import logo from '../../../../assets/Images/logo.png'
+import { useState } from "react"
+import { useForm } from "react-hook-form"
+import { AUTH_URL, axiosInstance } from "../../../../services/EndPoints"
+import { toast } from "react-toastify"
+
 export default function ChangePassword() {
+  const [isPasswordValid, setIsPasswordValid] = useState(false)
+  const [isNewPassValid, setIsNewPassValid] = useState(false)
+  const [isConfirmPassValid, setIsConfirmPassValid] = useState(false)
+  const {register, formState:{errors}, handleSubmit} = useForm()
+
+  const onSumbit  = async (data:any) =>{
+
+    try {
+      await axiosInstance.put(AUTH_URL.CHANGE_PASSWORD, data);
+      toast.success("Login is successfully");
+      
+    } catch (error:any) {
+      toast.error(error.response?.data?.message);
+    }
+  
+  
+  }
   return (
-    <div>
-      <h1>Change Password</h1>
+    <>
+      <div className='container-fluid vh-100'>
+ <div className='row justify-content-center align-items-center'>
+    <div className=' col-md-10 col-lg-6 m-5'>
+      <div className='text-center'>
+        <img className="img-fluid" src={logo} alt="" />
+      </div>
+     <div className={`${styles["form"]}`}>
+     <div className='py-3 px-4'>
+     <div className='py-3 px-4'>
+     <p className='text-white m-0 pt-3'>welcome to PMS</p>
+     <h2 className="m-0">Change Password</h2>
+    <div className="ps-1" style={{marginTop:"-12px"}}>
+    <svg width="16" height="4" viewBox="0 0 16 4"
+      fill="none" xmlns="http://www.w3.org/2000/svg">
+      <rect width="16" height="4" fill="#EF9B28"/>
+      </svg>
     </div>
+
+     </div>
+     <form onSubmit={handleSubmit(onSumbit)} className='d-flex flex-column py-3 px-4 '>
+       <div className={`${styles["input-container"]}`}>
+       <input {...register("oldPassword", {
+        required:"Old Password Is Required"
+       })} type={isPasswordValid? "text" : "password" } 
+       placeholder='Enter your Old Password'/>
+       <i 
```

Comment:
We should avoid having onClicks in non-interactive elements like icons. We should make this an icon button. I also recommend creating a password input component that will contain all the logic instead of repeating it in multiple places.

Reply:
We can add here a span description the action to be performed by the button and use sr-only class as we did in the session.